docs/source/details/backendconfig.rst

```diff
@@ -0,0 +1,47 @@
+.. _backendconfig
+
+Backend-specific configuration
+==============================
+
+While the openPMD API intends to be a backend-*independent* implementation of the openPMD standard, it is often necessary to pass configuration parameters to the concrete backend in use. Some backends recognize a number of configuration options via environment variables (see the backends' documentations), the preferred way to do so, however, *– as far as it is implemented –* is via a JSON-formatted string.
+
+The fundamental structure of this JSON configuration string is given as follows:
+
+.. literalinclude:: config_layout.json
```

Comment:
Follow-up PR: can we add a [schema](https://json-schema.org/) for this?

Reply:
Probably a good idea, yeah